### ISSUE TITLE
Should anucleate and nucleated be disjoint?

### ISSUE THREAD

**Issue by bvarner-ebi** [obophenotype/cell-ontology#1393]
Could someone comment on whether the classes indicated below should be made disjoint? 
If not, could you explain the reasoning? 

PATO_0001405 anucleate disjoint with PATO_0002505 nucleated  
PATO_0001407 mononuclate disjoint with PATO_0001908 | multinucleate

Additionally, should 'nucleated' be renamed to 'nucleate' to be in line with the other labels (e.g., anucleate, mononucleate, multinucleate, binucleate (all without a 'd' at the end))?

@dosumis, @paolaroncaglia, is there are particular person I should mention in this issue if it is suitable for additional review/follow up?

**Comment by paolaroncaglia** [comment]
Hi @bvarner-ebi ,
Some quick thoughts:
The topic was discussed previously, please check out related issues (pro tip: in the [main issue page](https://github.com/obophenotype/cell-ontology/issues), add "anucleate" in the filter textbox, and inspect open and closed issues; works with CL IDs too, though IDs make for stricter search parameters);
The same cell type may be nucleated and then become anucleated in time (think red blood cells), or be mononucleated and then become multinucleate (think cell division uncoupled from karyokinesis), and I'm not sure if disjointness axioms would get in the way of representing those transitions. As I said, quick thoughts; I'd suggest to review previous threads, and if doubts or concerns persists, please note them here. I'll leave this ticket open, and assign it to you to add your comments. Thank you!

**Comment by paolaroncaglia** [comment]
(@bvarner-ebi to clarify - I'm assigning the ticket to you so you can "own" the discussion. Edits to such high-level terms should be discussed with other editors, so please don't feel like you have to solve the puzzle all by yourself :-) but it's good practice to "lead" or at least follow up on tickets we editors open ourselves, otherwise they tend to fall through the cracks. Additional or alternative assignees may be added as the discussion evolves.)

**Comment by bvarner-ebi** [comment]
Thank you, @paolaroncaglia. Your advice and direction make sense to me. 

I had not thought about the temporal aspect. However, at any specific moment in time, I don't think a cell can be nucleate and anucleate at the same time. I'll have a look at the previous threads.

For the RBC example, I do see there are separate classes for enucleate erythrocyte (should this should be **a**nucleate erythrocyte?) and nucleate erythrocyte. An 'enucleate' erythrocyte could never be a nucleate erythrocyte at the same time, so I think the disjoint axiom would make sense, but I will investigate more.

**Comment by addiehl** [comment]
Well you can always add it in a branch and see if the reasoning breaks. It is good to be be careful and conservative when adding such a strong axiom.

**Comment by dosumis** [comment]
PATO should have 

```
bearer_of some nucleated disjointWith bearer_of some anucleate
multinucleate subClassOf nucleated
```

That should be sufficient for the check that you want. We should check whether these axioms makes it through the (recently revised) import process 

It would be good to add these details into the [relations guide](https://github.com/obophenotype/cell-ontology/blob/master/docs/relations_guide.md)


We deal with cells that transition between nucleated and anucleate by making subclasses and adding the assertions there, e.g. 

![image](https://user-images.githubusercontent.com/112839/147094615-558b0d08-5d84-43c5-b269-f1d74e9ffe63.png)

**Comment by dosumis** [comment]
I've just checked.  The disjointness axioms are not in PATO, but ~equivalent axioms are already in CL:

![image](https://user-images.githubusercontent.com/112839/147096078-4ddfa0c0-ec53-4f75-9383-c85f7308f117.png)

(I may have misremembered how this is recorded.)

@bvarner-ebi  - The pattern here is like the one we used for making red and white wine disjoint in the tutorial exercises vs making the colors red and white disjoint.

**Comment by bvarner-ebi** [comment]
Thanks for reviewing, @dosumis!

When it comes to imported terms in CL, do we have the latitude to make those changes in CL or do we need additional communication and sign-off?

I did go ahead and make the disjoints to the PATO classes and changed 'nucleated' to 'nucleate' in a branch (issue#1393-nucleateDisjoint). I did not find any errors after running the reasoner, but did notice many changes to the merged_import.owl file. Do you know why these changes appeared and how I should test what implications they would have?

**Comment by bvarner-ebi** [comment]
Spoke with @dosumis. He explained that edits to terms in CL that come from PATO would be overwritten with the next import, so the changes, if any, should happen in PATO. If we'd like to pursue the changes mentioned in this ticket, it was suggested to add it to one of the meeting agendas to get more feedback. I deleted my branch ((issue#1393-nucleateDisjoint).
We also discussed developing a CL style guide that could help address similar questions about word choice and modelling.
